### DISCUSSION TITLE
Vuetify 3 - component v-text-field not working

### DISCUSSION THREAD

**Discussion by pauloasantos** [vuetifyjs/vuetify#13780] [Q&A]
Hello,

The v-text-field component is not in version 3?

I'm trying using with VueJs 3 + Vuetify 3.0.0.alpha.5.

![image](https://user-images.githubusercontent.com/13980664/121387550-2235ff80-c921-11eb-85ce-7d270ea40c18.png)

Thanks.

**Comment by jacekkarczmarczyk** [comment]
It's not converted yet

**Reply by invex0811** [reply]
But you do not know how soon more functionality will appear on vuetifi 3, it is just written that resis is somewhere in q3, it is interesting that new functions will be added a little before the release?

**Reply by glen-84** [reply]
VTextField is currently in development (#13996), and _may_ be released in one of the next few alphas.
